dotnet/src/SemanticKernel.Skills/Skills.Memory.Sqlite/Skills.Memory.Sqlite.csproj

```diff
@@ -14,7 +14,7 @@
   <PropertyGroup>
     <!-- NuGet Package Settings -->
     <PackageId>Microsoft.SemanticKernel.Skills.Memory.Sqlite</PackageId>
-    <Title>Semantic Kernel - SQLite Connector</Title>
+    <Title>Semantic Kernel - SQLite Skill</Title>
```

Comment:
This one isn't actually a skill -- there have been other convos about how this should be in a different package.

Reply:
Should I just leave this alone?  Or what would be the best title here?

Reply:
I'll keep these changes for another PR to avoid crossing the streams